pandas/tests/frame/test_timestamps.py

```diff
@@ -0,0 +1,50 @@
+""" test DataFrame-boxed versions of the scalar Timestamp """
+from datetime import datetime
+
+import numpy as np
+
+import pandas.util.testing as tm
+from pandas.tseries import offsets
+
+from pandas.compat import lrange
+from pandas import (Timestamp, date_range,
+                    Series, DataFrame, DatetimeIndex)
+
+
+class TestDataFrameTimestamps(object):
```

Comment:
so we already have test_timeseries.py, is there a reason you are creating a new module here?

Reply:
Not really, will move.